src/main/java/ch/njol/skript/util/slot/EquipmentSlot.java

```diff
@@ -185,6 +185,21 @@ public void setItem(final @Nullable ItemStack item) {
 			PlayerUtils.updateInventory((Player) e.getHolder());
 	}
 	
+	@Override
+	public int getAmount() {
+		ItemStack item = slot.get(e);
+		return item != null ? item.getAmount() : 0;
+	}
+	
+	@Override
+	public void setAmount(int amount) {
+		ItemStack item = slot.get(e);
+		if (item != null)
+			item.setAmount(amount);
+		if (e.getHolder() instanceof Player)
+			PlayerUtils.updateInventory((Player) e.getHolder());
```

Comment:
Is updating the inventory needed?

Reply:
I was going off of the setItem method, just made an assumption.
I can test without. Will report back

Reply:
Ok I removed it, tested, it works. So I will remove and push shortly!

Reply:
Pushed and ready to go!